source/jormungandr/jormungandr/interfaces/v1/serializer/status.py

```diff
@@ -152,9 +152,8 @@ class StreetNetworkSerializer(OutsideServiceCommon):
     modes = StringListField(display_none=True)
     timeout = MethodField(schema_type=float, display_none=False)
     backend_class = MethodField(schema_type=str, display_none=False)
-    matrix_type = MethodField(schema_type=str, display_none=False)
+    lapse_time_matrix_to_retry = MethodField(schema_type=str, display_none=False)
```

Comment:
```suggestion
    lapse_time_matrix_to_retry = Field(schema_type=str, display_none=True)
```

Reply:
If I use Field, the field will be always displayed. However, this is a optional field and we want only a display in case of a value

Reply:
:+1: